### ISSUE TITLE
Integrate World Map of Letters

### ISSUE THREAD

**Issue by samimak37** [dhmit/computation_hist#172]
Develop a map that allows users to sort for letters from a certain place (i.e. letters from professors at other universities, areas, etc.)

**Comment by mscuthbert** [comment]
Is @mgallegos3015 someone to loop into this?  (were you working on mapping?)

**Comment by mgallegos3015** [comment]
Yeah, I'm definitely working on mapping at the moment, and this sounds very interesting!
